mlir/include/mlir-c/Dialect/Rock.h

```diff
@@ -88,8 +88,9 @@ void mlirRockTuningTableDestroy(MlirRockTuningTable table);
 // the best performing tuning parameter to simplify the underlying
 // implementation, which can be revisited in the future.
 MLIR_CAPI_EXPORTED
-bool mlirRockTuningUpdateTable(MlirRockTuningTable perfTable, char *probCStr,
-                               char *perfCStr, float time);
+bool mlirRockTuningUpdateTable(MlirRockTuningTable perfTable,
+                               const char *probCStr, const char *perfCStr,
```

Comment:
Compiler warning came from passing a const-char-* to a char-*.  This function passes it to another const-char-* anyway, just add the const here.